cmd/argocd/commands/project.go

```diff
@@ -677,9 +777,9 @@ func NewProjectListCommand(clientOpts *argocdclient.ClientOptions) *cobra.Comman
 			projects, err := projIf.List(context.Background(), &project.ProjectQuery{})
 			errors.CheckError(err)
 			w := tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)
-			fmt.Fprintf(w, "NAME\tDESCRIPTION\tDESTINATIONS\n")
+			fmt.Fprintf(w, "NAME\tDESCRIPTION\tDESTINATIONS\tSOURCES\tCLUSTER RESOURCE WHITELIST\tNETWORK RESOURCE BLACKLIST\n")
```

Comment:
`NETWORK RESOURCE BLACKLIST` should read `NAMESPACE RESOURCE BLACKLIST`

Also I think these should be hyphenated to indicate it's a single column: `CLUSTER-RESOURCE-WHITELIST` and `NAMESPACE-RESOURCE-BLACKLIST`